api/AudioParam.json

```diff
@@ -0,0 +1,637 @@
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": "14"
+            },
+            "chrome_android": {
+              "version_added": "14"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "25"
+            },
+            "firefox_android": {
+              "version_added": "26"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "ie_mobile": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "15"
+            },
+            "opera_android": {
+              "version_added": "15"
+            },
+            "safari": {
+              "version_added": "6"
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "cancelAndHoldAtTime": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/AudioParam/cancelAndHoldAtTime",
+          "support": {
+            "webview_android": [
+              {
+                "version_added": "57"
+              },
+              {
+                "version_added": true,
+                "version_removed": "56",
+                "notes": "Was implemented as <code>cancelValuesAndHoldAtTime()</code> before Chrome 57"
+              }
+            ],
+            "chrome": [
+              {
+                "version_added": "57"
+              },
+              {
+                "version_added": true,
+                "version_removed": "56",
+                "notes": "Was implemented as <code>cancelValuesAndHoldAtTime()</code> before Chrome 57"
+              }
+            ],
+            "chrome_android": [
+              {
+                "version_added": "57"
+              },
+              {
+                "version_added": true,
+                "version_removed": "56",
+                "notes": "Was implemented as <code>cancelValuesAndHoldAtTime()</code> before Chrome 57"
+              }
+            ],
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "ie_mobile": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "44"
```

Comment:
Did Opera supported also the "alternate_name": "cancelValuesAndHoldAtTime" for some versions?